.eslintcache

```diff
@@ -1 +1 @@
-[{"X:\\Projects\\budget-application\\budget-application\\src\\index.tsx":"1","X:\\Projects\\budget-application\\budget-application\\src\\stories\\Page.js":"2"},{"size":146,"mtime":1608056206668,"results":"3","hashOfConfig":"4"},{"size":2503,"mtime":1608056207038,"results":"5","hashOfConfig":"4"},{"filePath":"6","messages":"7","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"72oxir",{"filePath":"8","messages":"9","errorCount":0,"warningCount":2,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"X:\\Projects\\budget-application\\budget-application\\src\\index.tsx",[],"X:\\Projects\\budget-application\\budget-application\\src\\stories\\Page.js",["10","11"],{"ruleId":"12","severity":1,"message":"13","line":7,"column":21,"nodeType":"14","messageId":"15","endLine":7,"endColumn":70},{"ruleId":"12","severity":1,"message":"16","line":7,"column":22,"nodeType":"17","messageId":"18","endLine":7,"endColumn":66},"@typescript-eslint/explicit-module-boundary-types","Missing return type on function.","ArrowFunctionExpression","missingReturnType","Object pattern argument should be typed.","ObjectPattern","missingArgTypeUnnamed"]
\ No newline at end of file
+[{"C:\\Users\\Idan Levian\\Desktop\\Projects\\budget-application\\src\\style\\global\\global.styles.tsx":"1"},{"size":400,"mtime":1608234323719,"results":"2","hashOfConfig":"3"},{"filePath":"4","messages":"5","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"bntxwc","C:\\Users\\Idan Levian\\Desktop\\Projects\\budget-application\\src\\style\\global\\global.styles.tsx",[]]
```

Comment:
could add it to gitignore

Reply:
For some reason the gitignore not ignore this file, so i remove his line from gitignore.
I will check more about it and fix it !